Bookish/Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -42,4 +42,15 @@
     </table>
 </div>
 
+<div class="padded-top btn-group" role="group">
+    <a type="button" class="btn btn-outline-dark @Model.GetDisabledClass(false)" 
+       asp-action="Catalogue" 
+       asp-route-pageNumber="@(Model.PageNumber - 1)"
+       asp-route-searchTerm="@Model.SearchTerm">Previous</a>
+    <a type="button" class="btn btn-outline-dark disabled">@Model.PageNumber</a>
+    <a type="button" class="btn btn-outline-dark @Model.GetDisabledClass(true)" 
+       asp-action="Catalogue" 
+       asp-route-pageNumber="@(Model.PageNumber + 1)"
+       asp-route-searchTerm="@Model.SearchTerm">Next</a>
```

Comment:
I would expect this to show either:
1. A next and previous button + a text field saying "Page x of y", or
2. A next and previous button + a button for each page, e.g. if there are three pages then a 1, 2 and 3 button, all enabled (pressing the button for the current page will just reload it), but with some styling showing which page you're currently on.

Does that make sense? 🙂